packages/browser/src/Session.ts

```diff
@@ -120,16 +115,19 @@ export async function silentlyAuthenticate(
     // on incoming redirect. This way, the user is eventually redirected back
     // to the page they were on and not to the app's redirect page.
     window.localStorage.setItem(KEY_CURRENT_URL, window.location.href);
-    await clientAuthn.login({
-      sessionId,
-      prompt: "none",
-      oidcIssuer: storedSessionInfo.issuer,
-      redirectUrl: storedSessionInfo.redirectUrl,
-      clientId: storedSessionInfo.clientAppId,
-      clientSecret: storedSessionInfo.clientAppSecret,
-      tokenType: storedSessionInfo.tokenType ?? "DPoP",
-      inIframe: options.inIframe,
-    });
+    await clientAuthn.login(
+      {
+        sessionId,
+        prompt: "none",
+        oidcIssuer: storedSessionInfo.issuer,
+        redirectUrl: storedSessionInfo.redirectUrl,
+        clientId: storedSessionInfo.clientAppId,
+        clientSecret: storedSessionInfo.clientAppSecret,
+        tokenType: storedSessionInfo.tokenType ?? "DPoP",
+        inIframe: options.inIframe,
+      },
+      getDefaultSession()
```

Comment:
```suggestion
      this
```
You don't want to necessarily use the default session, but rather the current one, be it the default or one that has been created manually by the app.

Reply:
"this" doesn't work in this instance because it is outside of the Session class... is there a method to get the default session alongside the manually created one, or do we just need to pass the session into silentlyAuthenticate as a param??